package/yast2-crowbar.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Tue Nov 21 10:11:58 UTC 2016 - rarcea@suse.com
+
+- Update SLE versions to SP2 for Cloud 7
+- 3.1.20
```

Comment:
After discussing with @jsuchome, can you bump to version 3.2.0? We'll keep 3.1.x for Cloud 6.